backend/src/main/java/ca/bc/gov/hlth/hnweb/config/WebClientConfig.java

```diff
@@ -159,24 +131,57 @@ public WebClient rapidWebClient() throws HNWebException {
 
 	private SslContext getSSLContext(String url, Resource certFile, String certPassword) throws HNWebException {
 
-		try {
-			KeyStore keyStore = KeyStore.getInstance(KEY_STORE_TYPE_PKCS12);
-			keyStore.load(new FileInputStream(certFile.getFile()), certPassword.toCharArray());
+		return getSSLContext(url, certFile, certPassword, null);
+	}
+	
+	private SslContext getSSLContext(String url, Resource certFile, String certPassword, Resource trustFile) throws HNWebException {
 
-			KeyManagerFactory keyManagerFactory = KeyManagerFactory.getInstance(KEY_MANAGER_FACTORY_TYPE_SUN_X509);
-			keyManagerFactory.init(keyStore, certPassword.toCharArray());
+		try {
+			KeyManagerFactory keyManagerFactory = createKeyManagerFactory(certFile, certPassword);			
 			
+			TrustManagerFactory trustManagerFactory = null;
+			if (trustFile != null) {
+				trustManagerFactory = createTrustManagerFactory(trustFile, HCIM_CA_CERT);				
```

Comment:
For this to be truly re-usable I think the trustedCertAlias should be a method parameter as well.

Reply:
Added parameter for trust alias to make it re-usable